app/controllers/tests_controller.rb

```diff
@@ -1,11 +1,23 @@
 class TestsController < Simpler::Controller
 
   def index
-    @time = Time.now
+    status 200
+    set_custom_headers ({'Custom-Header' => 'custom-value', 'Custom-Header-2' => 'custom-value-2'})  	
```

Comment:
Я не про те пробелы говорил.
```suggestion
    set_custom_headers({ 'Custom-Header' => 'custom-value', 'Custom-Header-2' => 'custom-value-2' })
```